src/tests/test_pkcs11_high_level.cpp

```diff
@@ -1342,14 +1342,14 @@ Test::Result test_pkcs11_hmac_drbg()
    Test::Result result("PKCS11 HMAC_DRBG using PKCS11_RNG");
    TestSession test_session(true);
 
-   HMAC_DRBG drbg(MessageAuthenticationCode::create("HMAC(SHA-512)").release(), new PKCS11_RNG(test_session.session()));
+   HMAC_DRBG drbg("SHA-512", 0);
```

Comment:
No argument, but how are you going to do that? HMAC_DRBG does not have the appropriate constructors any more.

HMAC_DRBG needs to be restored to what it was last September to provide that capability again.